### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Sick-Carlito**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,971 @@
...
+                        </article>
+                        <article>
+                            <div class="wrap">
+                                <a><img src="images/john-bolton.jpg" class="img-fluid" alt="Responsive image"><span class="pic-on">Politics</span></a>
+                            </div>
+                            <div class="article-info">
+                                <h3>John Bolton Slams Trump's Foreign Policy at Private Lunch: Report</h3>
+                                <div class="summary">The former national security adviser also reportedly asserted that President Donald Trump's negotiations with Iran and North Korea are "doomed to failure."</div>
+                            </div>
+                            <!--<div class="related">
```
Comment (whiz25) [inline index.html]: remove this comment

**Comment by Sick-Carlito** [comment]
Thanks guys !!

File: index.html (hunk 1 above)
Comment (Sick-Carlito) [reply index.html]: Hey bro, what's your official name? I can't find you in the TSEs list.

Comment (whiz25) [reply index.html]: Martin Afani

**Pull request merged**